src/js/Parser.js

```diff
@@ -0,0 +1,53 @@
+import { Station } from './Station';
+import { SubwayLine } from './SubwayLine';
+import { Subway } from './Subway';
+
+export function parse(data) {
+    let currentData = data;
```

Comment:
you do not need to store data to local variable you can just pass it to you function. after thatyou even do not need convertToSubway function you can move it's content to parse function